src/main/java/org/wso2/carbon/connector/util/Error.java

```diff
@@ -0,0 +1,69 @@
+/*
+ *  Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.wso2.carbon.connector.util;
+
+/**
+ * Enum representing error codes and messages for the Microsoft Azure storage connector.
+ */
+public enum Error {
+
+    CONNECTION_ERROR("710001", "MS_AZURE_DATALAKE_GEN2:CONNECTION_ERROR"),
```

Comment:
Shall we declare a constant for "MS_AZURE_DATALAKE_GEN2:" part and use it here?

Ex: 
public static final String ERROR_PREFIX = "MS_AZURE_DATALAKE_GEN2:";

INVALID_CONFIGURATION("710002", ERROR_PREFIX + "INVALID_CONFIGURATION"),